tsfile/src/test/java/org/apache/iotdb/tsfile/file/metadata/utils/Utils.java

```diff
@@ -1,209 +1,245 @@
-        for (int i = 0; i < metadata1.getChunkGroupMetaDataList().size(); i++) {
-          Utils.isChunkGroupMetaDataEqual(metadata1.getChunkGroupMetaDataList().get(i),
-              metadata1.getChunkGroupMetaDataList().get(i));
-        }
-      }
-    }
-  }
-
-  public static void isChunkGroupMetaDataEqual(ChunkGroupMetaData metadata1,
-      ChunkGroupMetaData metadata2) {
-    if (Utils.isTwoObjectsNotNULL(metadata1, metadata2, "ChunkGroupMetaData")) {
-      assertTrue(metadata1.getDeviceID().equals(metadata2.getDeviceID()));
-
-      if (Utils
-          .isTwoObjectsNotNULL(metadata1.getChunkMetaDataList(), metadata2.getChunkMetaDataList(),
-              "Timeseries chunk metadata list")) {
-        assertEquals(metadata1.getChunkMetaDataList().size(),
-            metadata2.getChunkMetaDataList().size());
-        for (int i = 0; i < metadata1.getChunkMetaDataList().size(); i++) {
-          Utils.isTimeSeriesChunkMetadataEqual(metadata1.getChunkMetaDataList().get(i),
-              metadata1.getChunkMetaDataList().get(i));
-        }
-      }
-    }
-  }
-
-  public static void isTsDeviceMetadataIndexEqual(TsDeviceMetadataIndex index1,
-      TsDeviceMetadataIndex index2) {
-    if (Utils.isTwoObjectsNotNULL(index1, index2, "TsDeviceMetadataIndex")) {
-      assertEquals(index1.getOffset(), index2.getOffset());
-      assertEquals(index1.getLen(), index2.getLen());
-      assertEquals(index1.getStartTime(), index2.getStartTime());
-      assertEquals(index1.getEndTime(), index2.getEndTime());
-    }
-  }
-
-  public static void isFileMetaDataEqual(TsFileMetaData metadata1, TsFileMetaData metadata2) {
-    if (Utils.isTwoObjectsNotNULL(metadata1, metadata2, "File MetaData")) {
-      if (Utils.isTwoObjectsNotNULL(metadata1.getDeviceMap(), metadata2.getDeviceMap(),
-          "Delta object metadata list")) {
-
-        Map<String, TsDeviceMetadataIndex> deviceMetadataMap1 = metadata1.getDeviceMap();
-        Map<String, TsDeviceMetadataIndex> deviceMetadataMap2 = metadata2.getDeviceMap();
-        assertEquals(deviceMetadataMap1.size(), deviceMetadataMap2.size());
-
-        for (String key : deviceMetadataMap1.keySet()) {
-          Utils.isTsDeviceMetadataIndexEqual(deviceMetadataMap1.get(key),
-              deviceMetadataMap2.get(key));
-        }
-      }
-
-      if (Utils
-          .isTwoObjectsNotNULL(metadata1.getMeasurementSchema(), metadata2.getMeasurementSchema(),
-              "Timeseries metadata list")) {
-        assertEquals(metadata1.getMeasurementSchema().size(),
-            metadata2.getMeasurementSchema().size());
-        for (Map.Entry<String, MeasurementSchema> entry : metadata1.getMeasurementSchema()
-            .entrySet()) {
-          entry.getValue().equals(metadata2.getMeasurementSchema().get(entry.getKey()));
-        }
-      }
-
-      assertEquals(metadata1.getCurrentVersion(), metadata2.getCurrentVersion());
-      assertEquals(metadata1.getCreatedBy(), metadata2.getCreatedBy());
-    }
-  }
-}

```

Comment:
formated?

Reply:
Maybe? I just add two method, isPageHeaderEqual() and isStatisticsEqual().

Reply:
restore this class and re-add your function, otherwise it can not pass apache license check